src/bridge/cockpitpcpmetrics.c

```diff
@@ -1096,8 +1096,8 @@ my_pmParseUnitsStr(const char *str, pmUnits *out, double *multiplier)
                                               [PM_TIME_MIN]  = 60,
                                               [PM_TIME_HOUR] = 3600 };
         // guaranteed by __pmParseUnitsStrPart; ensure in-range array access
-        assert (dividend.scaleTime >= PM_TIME_NSEC && dividend.scaleTime <= PM_TIME_HOUR);
-        assert (divisor.scaleTime >= PM_TIME_NSEC && divisor.scaleTime <= PM_TIME_HOUR);
+        assert (dividend.scaleTime <= PM_TIME_HOUR);
+        assert (divisor.scaleTime <= PM_TIME_HOUR);
```

Comment:
@mvollmer I think you should be using G_STATIC_ASSERT instead of a runtime assert if you want to assert the value of a constant.